### PR TITLE
docs: update README.md

### PR DESCRIPTION
**Pull request by AlexWZ0129**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -11,8 +13,10 @@
 | [@cennznet/util](packages/util/README.md)     | cennznet specific utility functions                         |
 | [@cennznet/types](packages/types/README.md)   | cennznet specific type definitions                                |
 
-# How to contribute
+# Cennznet runtime module libraries
```
Comment (AlexWZ0129) [inline README.md]: do we want this?

File: README.md [hunk 2]
```diff
@@ -2,6 +2,8 @@
 
 > The Cennznet JavaScript API library for browsers and Node.js.
 
+> Please read the [documentation](https://cennznetdocs.com/api/latest/tutorials/0_Overview.md) for more.
```
Comment (ianhe8x) [inline README.md]: don't use the reference syntax

File: README.md (hunk 1 above)
Comment (ianhe8x) [reply README.md]: We can have this section here, doesn't hurt.

File: README.md (hunk 2 above)
Comment (AlexWZ0129) [reply README.md]: yep, fixed

**Comment by ianhe8x** [comment]
lgtm

**Pull request merged**